src/_guides/language/analysis-options.md

```diff
@@ -376,11 +376,48 @@ linter:
 ```
 
 {{site.alert.note}}
-  Due to YAML restrictions, 
+  Due to YAML restrictions,
   **you can't mix list and key-value syntax in the same `rules` entry.**
-  You can, however, use the other syntax for rules in an included file.
+  You may use the other syntax for rules in an included file.
```

Comment:
Consider reverting or changing this, since we [avoid "may"](https://developers.google.com/style/word-list#may).